### PR TITLE
Install SwiftLint & SwiftFormat

### PR DESCRIPTION
**Pull request by fand**
## Changes
- Install SwiftLint & SwiftFormat
  - Add settings to run them on build
- Fix lint errors 

The diff is large but most changes are just small fix (I prefer using default lint rules rather than customizing them)
The biggest one is splitting rendering process of CommandDetailSettingsViewController.

### CODE REVIEW

File: ZIGSIMPlus/Models/InAppPurchaseFacade.swift [hunk 1]
```diff
@@ -59,16 +59,16 @@ class InAppPurchaseFacade: NSObject {
...
         }
-        
+
         return id
     }
 }
 
 extension InAppPurchaseFacade: SKProductsRequestDelegate {
-    func productsRequest(_ request: SKProductsRequest, didReceive response: SKProductsResponse) {
+    func productsRequest(_: SKProductsRequest, didReceive response: SKProductsResponse) {
```
Comment (nozomuplus) [inline ZIGSIMPlus/Models/InAppPurchaseFacade.swift]: Will this delegate method called properly? 
I'm not sure.
If YES, it's OK.

File: ZIGSIMPlus/Models/NetworkAdapter.swift [hunk 2]
```diff
@@ -67,8 +73,8 @@ public class NetworkAdapter {
             switch tcpClient.connect(timeout: 1) {
             case .success:
                 break
-            case .failure(let e):
-                error = e
+            case let .failure(err):
+                error = err
```
Comment (nozomuplus) [inline ZIGSIMPlus/Models/NetworkAdapter.swift]: You can allow short name for local variables.
But if you want to follow the strict rule, it's OK.

File: ZIGSIMPlus/Models/InAppPurchaseFacade.swift (hunk 1 above)
Comment (fand) [reply ZIGSIMPlus/Models/InAppPurchaseFacade.swift]: uh, I think it doesn't work... will fix them.
SwiftLint can't detect delegate methods .https://github.com/nicklockwood/SwiftFormat/issues/83

Comment (nozomuplus) [reply ZIGSIMPlus/Models/InAppPurchaseFacade.swift]: I tested, and they seem to be called!

File: ZIGSIMPlus/Views/CommandOutputViewController.swift [hunk 3]
```diff
@@ -6,18 +6,18 @@
...
-    @IBOutlet weak var textField: UITextView!
-    @IBOutlet weak var touchArea: UIView!
-    @IBOutlet weak var textPreview: UIView!
-    @IBOutlet weak var imagePreview: UIView!
-    @IBOutlet weak var togglePreviewModeButton: UIBarButtonItem!
+    @IBOutlet var textField: UITextView!
+    @IBOutlet var touchArea: UIView!
+    @IBOutlet var textPreview: UIView!
+    @IBOutlet var imagePreview: UIView!
+    @IBOutlet var togglePreviewModeButton: UIBarButtonItem!
```
Comment (nozomuplus) [inline ZIGSIMPlus/Views/CommandOutputViewController.swift]: IBOutlets should be weak except for special cases, shouldn't they?
https://qiita.com/chocovayashi/items/a96adc1356b7c45524b7

Comment (fand) [reply ZIGSIMPlus/Views/CommandOutputViewController.swift]: Apple has changed the opinion about holding views; now they recommend strong refs except for refs to views back up the view hierarchy.
https://stackoverflow.com/a/31395938

File: ZIGSIMPlus/Models/NetworkAdapter.swift (hunk 2 above)
Comment (fand) [reply ZIGSIMPlus/Models/NetworkAdapter.swift]: yea I wanted to leave the rules default as long as I can.

**Comment by nozomuplus** [comment]
What's the timing of running SwiftFormat and SwiftLint?

**Comment by fand** [comment]
> What's the timing of running SwiftFormat and SwiftLint?

They run on build phase. See https://github.com/1-10/ZIGSIMPlus_iOS_Public/pull/96/files#diff-523a922ef4d1a2899a58aff29cbe93f2R734 and https://github.com/1-10/ZIGSIMPlus_iOS_Public/pull/96/files#diff-523a922ef4d1a2899a58aff29cbe93f2R752

**Comment by nozomuplus** [comment]
Got it.
If you don't feel long, no problem.
If you feel it takes too long, consider running them at test or commit time.

File: ZIGSIMPlus/Views/CommandOutputViewController.swift (hunk 3 above)
Comment (nozomuplus) [reply ZIGSIMPlus/Views/CommandOutputViewController.swift]: Oh I didn't know that.
Thanks!

File: .swiftformat [hunk 4]
```diff
@@ -0,0 +1,2 @@
+--exclude Pods
+--stripunusedargs closure-only
```
Comment (nozomuplus) [inline .swiftformat]: :thumbsup:

**Comment by fand** [comment]
Thx ٩(ˊᗜˋ*)و

**Pull request merged**